vp/workflow/views.py

```diff
@@ -0,0 +1,52 @@
+from django.http import JsonResponse
+import networkx as nx
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+def open(request):
+
+    # Open filename specified
+    if request.GET.get('file'):
+        path = os.path.join(BASE_DIR, request.GET.get('file'))
```

Comment:
Same concern about writing to the application code directory, but I'd just leave this for now until we tackle upload/download-based load/save